mujoco_warp/_src/collision_hfield.py

```diff
@@ -100,20 +77,20 @@ def hfield_triangle_prism(
   hfieldid: int,
   hftri_index: int,
 ) -> wp.mat33:
-  """Returns the vertices of a triangular prism for a heightfield triangle.
+  """Returns triangular prism vertex information in compressed representation.
 
   Args:
-    geom_dataid: Array of geometry data IDs
-    hfield_adr: Array of heightfield addresses
-    hfield_nrow: Array of heightfield rows
-    hfield_ncol: Array of heightfield columns
-    hfield_size: Array of heightfield sizes
-    hfield_data: Array of heightfield data
-    hfieldid: Index of the height field geometry
-    hftri_index: Index of the triangle in the heightfield
+    geom_dataid (wp.array(dtype=int)): geom data ids
```

Comment:
I don't think we need to put the type annotation here, I'm not sure it helps the reader as it's located directly above in the function signature.

If it's important to you to add some metadata, maybe just the shape would suffice, so something like:

```
geom_dataid (ngeom,): id of geom's mesh/hfield; -1: none
hfield_adr (nhfield,): start address in hfield_data
```

Reply:
removed type annotations from docstrings in `collision_hfield.py` and created an issue to standardize the docstring style https://github.com/google-deepmind/mujoco_warp/issues/650.